01_fib/fibonacci.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(num int) {
+	num1, num2 := 0, 1
+
+	var result string
+	for i := 0; i < num; i++ {
+		result += strconv.Itoa(num1)
```

Comment:
Using `fmt.Fprintln(out, num1)` made `result` redundant, you can remove it.
I'm actually surprised it compiled without an unused variable error.

Reply:
Done